index.html

```diff
@@ -0,0 +1,539 @@
+          </div>
+        </div>
+        <div>
+          <div class="img-part">
+            <a href="#">
+              <img class="transformer" src="images/rik-schennink-100px-opt.png" alt="">
+            </a>
+          </div>
+          <div class="column-part">
+            <h3><a href="#"> Rik Schennink</a> wrote</h3>
+            <h1><a href="#"> Flip</a></h1>
+            <p> <span class="time-span"> March 10, 2020</span> — Whether you want to count down to an event, visualize a
+              fundraising campaign,
+              show a clock or a sales counter, Rik Schennink’s <a href="#"> open-source flip counter plugin</a> has got
+              you
+              covered.</p>
+          </div>
+        </div>
+        <div>
+          <div class="img-part">
+            <a href="#">
+              <img class="transformer" src="images/braid-opt.png" alt="">
+            </a>
+          </div>
+          <div class="column-part">
+            <h3><a href="#"> Braid</a> wrote</h3>
+            <h1><a href="#"> Vue Formulate</a></h1>
+            <p> <span class="time-span"> March 09, 2020</span> — Forms are often rather tedious to author. <a href="#">
+                Vue Formulate</a> provides a powerful and flexible API that helps developers handling forms with Vue.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="img-part">
+            <a href="#">
+              <img class="transformer" src="images/mark-boulton-100px-opt.png" alt="">
+            </a>
+          </div>
+          <div class="column-part">
+            <h3><a href="#"> Mark Boulton</a> wrote</h3>
+            <h1><a href="#"> Leading A Remote Design Team</a></h1>
+            <p> <span class="time-span"> March 6, 2020</span> — Leading a remote design team can be a bit daunting at
+              first. Mark Boulton shares some <a href="#"> simple principles </a>that help you get into the swing of
+              things quickly, so that you can start to become more productive and effective.</p>
+          </div>
+        </div>
+        <div>
+          <div class="img-part">
+            <a href="#">
+              <img class="transformer" src="images/jason-pamental-100px-opt.png" alt="">
+            </a>
+          </div>
+          <div class="column-part">
+            <h3><a href="#"> Jason Pamental</a> wrote</h3>
+            <h1><a href="#"> A Variable Fonts Primer</a></h1>
+            <p> <span class="time-span"> March 5, 2020</span> — With variable fonts, we can accomplish what used to
+              require several files with a single file and some CSS. The <a href="#"> “A Variable Fonts Primer”</a>
+              teaches you how variable fonts tick and how they can benefit UI design, accessibility, and long-form
+              reading.</p>
+          </div>
+        </div>
+        <a href="#">More links on Twitter → </a>
+      </div>
+    </div>
+  </section>
+  <section class="featured">
```

Comment:
This shouldn't have a red bg color, just the correspondent gray.

Reply:
Done

Reply:
Done!